src/C4PlayerInfoListBox.cpp

```diff
@@ -921,21 +921,25 @@ C4GUI::ContextMenu *C4PlayerInfoListBox::ClientListItem::OnContext(C4GUI::Elemen
 	C4Client *pClient = GetClient();
 	// create context menu
 	C4GUI::ContextMenu *pMenu = new C4GUI::ContextMenu();
+	// helper function
+	auto AddMenuItem = [&](const char *text, const char *description, auto callbackFn)
+	{
+		pMenu -> AddItem(LoadResStr(text), LoadResStr(description), C4GUI::Ico_None,
+			new C4GUI::CBMenuHandler<ClientListItem>(this, callbackFn));
+	};
 	// host options
 	if (Game.Network.isHost() && GetNetClient())
 	{
-		StdStrBuf strKickDesc(LoadResStr("IDS_NET_KICKCLIENT_DESC"));
-		pMenu->AddItem(LoadResStr("IDS_NET_KICKCLIENT"), strKickDesc.getData(), C4GUI::Ico_None,
-			new C4GUI::CBMenuHandler<ClientListItem>(this, &ClientListItem::OnCtxKick));
-		StdStrBuf strActivateDesc(LoadResStr("IDS_NET_ACTIVATECLIENT_DESC"));
-		pMenu->AddItem(LoadResStr(pClient->isActivated() ? "IDS_NET_DEACTIVATECLIENT" : "IDS_NET_ACTIVATECLIENT"),
-			strActivateDesc.getData(), C4GUI::Ico_None,
-			new C4GUI::CBMenuHandler<ClientListItem>(this, &ClientListItem::OnCtxActivate));
+		AddMenuItem("IDS_NET_KICKCLIENT", "IDS_NET_KICKCLIENT_DESC", &ClientListItem::OnCtxKick);
+		AddMenuItem(pClient->isActivated() ? "IDS_NET_DEACTIVATECLIENT" : "IDS_NET_ACTIVATECLIENT", "IDS_NET_ACTIVATECLIENT_DESC", &ClientListItem::OnCtxActivate);
 	}
 	// info
-	StdStrBuf strClientInfoDesc(LoadResStr("IDS_NET_CLIENTINFO_DESC"));
-	pMenu->AddItem(LoadResStr("IDS_NET_CLIENTINFO"), strClientInfoDesc.getData(), C4GUI::Ico_None,
-		new C4GUI::CBMenuHandler<ClientListItem>(this, &ClientListItem::OnCtxInfo));
+	AddMenuItem("IDS_NET_CLIENTINFO", "IDS_NET_CLIENTINFO_DESC", &ClientListItem::OnCtxInfo);
+	// mute / unmute
+	if (pClient->isMuted())
+		AddMenuItem("IDS_NET_UNMUTE", "IDS_NET_UNMUTE_DESC", &ClientListItem::OnCtxUnmute);
+	else
+		AddMenuItem("IDS_NET_MUTE", "IDS_NET_MUTE_DESC", &ClientListItem::OnCtxMute);
```

Comment:
The (still unpublished) style guidelines allow omitting braces only for ifs without else.
However this might be irrelevant due to other changes.